spaceshooter/spaceShooter.py

```diff
@@ -505,7 +505,8 @@ def update(self):
             pow = Pow(hit.rect.center)
             all_sprites.add(pow)
             powerups.add(pow)
-        newmob()        ## spawn a new mob
+        for m in range(int(1+score/3000)):
+            newmob()        ## spawn a new mob
```

Comment:
So, you have increased number of mobs at 100% of initial value every 3000 score. Maybe it will be better to make more smooth switch? 100% at a time is a lot, it can like "no mobs" and "a lot of mobs" transition, because of x2, while reaching every new barrier will give us +33%, +25% or total amount of mobs and it may be unnoticeable.